scripts/verify.ts

```diff
@@ -8,13 +8,10 @@ async function main() {
     throw Error(`no proxy entry for '${network.name}'`);
   }
 
-  // Verify implementation
-  const tables = (await ethers.getContractFactory("TablelandTables")).attach(
-    proxy
-  );
-  const impl = await upgrades.erc1967.getImplementationAddress(tables.address);
+  // Verify proxy and implementation
+  // This will also link the proxy to the implementation in the explorer UI
```

Comment:
I should have mentioned, this is new functionality in the verify method. no longer need to verify the impl and like it to the proxy using the block explorer UIs.